PasswordTestCases/PasswordVerifyFixture.cs

```diff
@@ -0,0 +1,39 @@
+using Problem;
+namespace PasswordTestCases
+{
+    public class PasswordVerifyFixture
+    {
+
+
+
+
+        [Theory]
+
+        [InlineData("absc1Dfew")]
```

Comment:
Remove positive inputs from negative test case and add new test case for positive scenarios.